diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter5/AvoidNestedFunctionsFixTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.cqfn.diktat.ruleset.chapter5
+
+import generated.WarningNames
+import org.cqfn.diktat.ruleset.rules.AvoidNestedFunctionsRule
+import org.cqfn.diktat.util.FixTestBase
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class AvoidNestedFunctionsFixTest : FixTestBase("test/paragraph5/nested_functions", ::AvoidNestedFunctionsRule) {
```

Comment:
Please add an example that your rule doesn't break functions with dependent variables.